test/unit/lib/challenge.test.ts

```diff
@@ -26,3 +27,130 @@ describe('Challenge Generation', (): void => {
     expect(Buffer.byteLength(challenge, 'base64')).toBe(8)
   })
 })
+
+describe('Signature Verification', (): void => {
+  // Repeat test with different keys
+  for (let index = 0; index < 4; index++) {
+    it('verifies correct signature - EC Key', async (): Promise<void> => {
+      const keyPair = crypto.generateKeyPairSync('ec', {
+        namedCurve: 'secp256k1', // Allowed by FIDO spec
+        publicKeyEncoding: {
+          type: 'spki', // Key infrasructure
+          format: 'pem' // Encoding format
+        },
+        privateKeyEncoding: {
+          type: 'pkcs8',
+          format: 'pem'
+        }
+      })
+
+      const challenge = 'Crypto Auth service Yay!'
```

Comment:
not sure but doesn't a valid base 64 string challenge have to end with '='?

Reply:
maybe just also try testing by using `generate` to gen challenges of different sizes (32, 64, etc.) and test verifySign with those?